_ml-commons-plugin/cluster-settings.md

```diff
@@ -292,6 +292,23 @@ plugins.ml_commons.enable_inhouse_python_model: false
 - Default value: false
 - Value range: [false, true]
 
+## Enable access control for connectors
+
+When set to `true`, requires an access key when making a connection to SageMaker or Hugging Face.
+
+### Setting
+
+```
+plugins.ml_commons.connector_access_control_enabled: true
+```
+
+###
+
+### Values
+
+- Default value: false
+- Value range: [false, true]
```

Comment:
It's not really a range but a set so I would say
```suggestion
- Valid values: `false`, `true`
```

Reply:
Fixed the other lines for this too.